src/variables-tree.js

```diff
@@ -7,8 +8,19 @@ export function buildVariablesTree(variables, query) {
     if (!query) {
       return true
     }
-    const normalizedQuery = query.toLowerCase()
-    return variable.name.toLowerCase().includes(normalizedQuery)
+    function nameMatches(variable, query) {
+      const normalizedName = variable.name.toLowerCase()
```

Comment:
No, not the same answer: it is [impossible](https://docs.python.org/2/reference/lexical_analysis.html#identifiers) (as long as the runtime is Python 2) for variable names to contain diacritics. Only a subset of the ASCII range is supported.